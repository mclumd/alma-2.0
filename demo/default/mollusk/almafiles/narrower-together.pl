rel(cephalopod, steve).
rel(nautilus, nancy).
rel(naked_nautilus, barry).

if(now(6), obs(quote(rel(cephalopod, sam)))).
