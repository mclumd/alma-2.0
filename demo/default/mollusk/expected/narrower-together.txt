rel(cephalopod, steve)
rel(mollusk, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
~rel(shell_bearer, steve)
rel(nautilus, nancy)
rel(cephalopod, nancy)
rel(mollusk, nancy)
abnormal(nancy, cephalopod, "rel(shell_bearer, nancy)")
rel(shell_bearer, nancy)
rel(naked_nautilus, barry)
rel(nautilus, barry)
rel(cephalopod, barry)
rel(mollusk, barry)
abnormal(barry, mollusk, "~rel(shell_bearer, barry)")
abnormal(barry, nautilus, "~rel(shell_bearer, barry)")
~rel(shell_bearer, barry)
rel(cephalopod, sam)
rel(mollusk, sam)
abnormal(sam, mollusk, "~rel(shell_bearer, sam)")
~rel(shell_bearer, sam)
